examples/chained_wizard.py

```diff
@@ -17,10 +17,11 @@
 # Put the Enthought library on the Python path.
 sys.path.append(os.path.abspath(r"..\..\.."))
```

Comment:
Again unrelated to this PR, but: do we still need this line?

Reply:
Yep, thought about removing it, but better in a different PR.